src/main/java/Duke.java

```diff
@@ -1,10 +1,163 @@
+import java.util.Scanner;
+
```

Comment:
Perhaps you could consider having a package / packages here to organise different types (e.g., classes, interfaces, enumerations, etc.) for easier management.